apps/finance/app/src/components/BalanceToken.js

```diff
@@ -74,7 +73,7 @@ const BalanceToken = ({
           `}
         >
           {convertedAmount >= 0
-            ? `$${formatTokenAmount(convertedAmount.toFixed(2))}`
+            ? `$${formatNumber(convertedAmount.toFixed(2))}`
```

Comment:
I believe the return value from `formatNumber` is an array, so we declare this as a variable above and use it here (also for the sake of readability).